lib/ProductOpener/Display.pm

```diff
@@ -854,6 +861,20 @@ sub init_request ($request_ref = {}) {
 	$request_ref->{admin} = $admin;
 	# TODO: remove the $admin global variable, and use $request_ref->{admin} instead.
 
+	if ($User{moderator}) {
+		$request_ref->{moderator} = 1;
+	}
+	else {
+		$request_ref->{moderator} = 0;
+	}
+
+	if ($User{pro_moderator}) {
+		$request_ref->{pro_moderator} = 1;
+	}
+	else {
+		$request_ref->{pro_moderator} = 0;
+	}
```

Comment:
```suggestion
	$request_ref->{moderator} = !!$User{moderator};
	$User{pro_moderator} = !!$request_ref->{pro_moderator};
```

Reply:
Here I wanted to set the value to 0 specifically, but we can use undef instead (!! 0 or !! undef seems to be equal to undef)